bn-expansions/src/EM/mortality-rates.ts

```diff
@@ -25,3 +25,86 @@ Note that all flat rate DMRs are applied regardless of the condition of the fish
 applied even to fish which may be in parts or otherwise obviously dead.
 
 */
+import { logbookExpansion } from "../base/em-rule-base";
+import { Catches, Disposition, GearTypes } from '@boatnet/bn-models';
+import { get, set } from 'lodash';
+
+const jp = require('jsonpath');
+
+// from IFQ Business Rules
+const discardMortalityRatesMap = {
+    SABL: {
+        [GearTypes.MidwaterTrawl]: 1,
+        [GearTypes.FishPot]: 20,
+        [GearTypes.HookAndLine]: 20,
+        // VerticalHookAndLineGear??
+        // OtherHookAndLineGear??
+        [GearTypes.GroundfishTrawlFootropeLessThan8]: 50,
+        [GearTypes.GroundfishTrawlGreaterThan8]: 50,
+    },
+    203: {
+        [GearTypes.MidwaterTrawl]: 1,
+        [GearTypes.FishPot]: 20,
+        [GearTypes.HookAndLine]: 20,
+        // VerticalHookAndLineGear??
+        // OtherHookAndLineGear??
+        [GearTypes.GroundfishTrawlFootropeLessThan8]: 50,
+        [GearTypes.GroundfishTrawlGreaterThan8]: 50,
+    },
+    LCOD: {
+        [GearTypes.MidwaterTrawl]: 100,
+        [GearTypes.FishPot]: 7,
+        [GearTypes.HookAndLine]: 7,
+        // VerticalHookAndLineGear??
+        // OtherHookAndLineGear??
+        [GearTypes.GroundfishTrawlFootropeLessThan8]: 50,
+        [GearTypes.GroundfishTrawlGreaterThan8]: 50
+    },
+    603: {
+        [GearTypes.MidwaterTrawl]: 100,
+        [GearTypes.FishPot]: 7,
+        [GearTypes.HookAndLine]: 7,
+        // VerticalHookAndLineGear??
+        // OtherHookAndLineGear??
+        [GearTypes.GroundfishTrawlFootropeLessThan8]: 50,
+        [GearTypes.GroundfishTrawlGreaterThan8]: 50
+    },
+    PHLB: {
+        [GearTypes.MidwaterTrawl]: 100,
+        [GearTypes.FishPot]: 18,
+        [GearTypes.HookAndLine]: 18,
+        // VerticalHookAndLineGear??
+        // OtherHookAndLineGear??
+       // [GearTypes.GroundfishTrawlFootropeLessThan8]: 50,
+       // [GearTypes.GroundfishTrawlGreaterThan8]: 50
+    },
+    101: {
+        [GearTypes.MidwaterTrawl]: 100,
+        [GearTypes.FishPot]: 18,
+        [GearTypes.HookAndLine]: 18,
+        // VerticalHookAndLineGear??
+        // OtherHookAndLineGear??
+       // [GearTypes.GroundfishTrawlFootropeLessThan8]: 50,
+       // [GearTypes.GroundfishTrawlGreaterThan8]: 50
+    }
+}
+
+export class discardMortalityRates implements logbookExpansion {
+    logbookExpansion(logbook: Catches) : Catches {
```

Comment:
Just making sure I understand this... you're using the logbookExpansion interface because you only need the one catch, as apposed to both the logbook and a review, but this can evaluate either a logbook or a review correct?

Reply:
Created just one interface that takes an ExpansionParameter object which can holds the currCatch and a bunch of optional fields including logbook, fishTickets ect. This still returns Catch and in this case will be run on logbook, review, and audit.